File: Intro/java/maxValue.java
class Source {
  public static double maxValue(double[] numbers) {
    double max = Double.NEGATIVE_INFINITY;
    for (double num : numbers) {
      if (num > max) {
        max = num;
      }
    }
    return max;
  }

  public static void run() {
    // this function behaves as `main()` for the 'run' command
    // you may sandbox in this function, but should not remove it
  }
}

File: Intro/csharp/MaxValue.cs
using System.Collections.Generic;

namespace IntroSolutions;

public static class MaxValue
{
    public static double Solve(IEnumerable<double> numbers)
    {
        double max = double.NegativeInfinity;
        foreach (double number in numbers)
        {
            if (number > max)
            {
                max = number;
            }
        }

        return max;
    }
}
